packages/alias/package.json

```diff
@@ -22,9 +22,9 @@
     "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
     "ci:test": "pnpm run test -- --verbose",
     "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
-    "lint:docs": "prettier --single-quote --arrow-parens avoid --trailing-comma none --write README.md",
+    "lint:docs": "prettier --write README.md",
     "lint:js": "eslint --fix --cache src test types --ext .js,.ts",
-    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
```

Comment:
the plugin should already be loaded automatically given it starts with `prettier-plugin-`

Reply:
ah didn't see this comment. in my tests with the plugins system I didn't find that to be the case. that's why we had the flag set all over the place.

Reply:
I'll double check the plugin is actually loaded

Reply:
Autoload doesn't work with pnpm. You can try specifying the plugin name in `.prettierrc`: `"plugins": "prettier-plugin-package"`.